latte/cs/template-inheritance.texy

```diff
@@ -361,7 +365,7 @@ Pomocí testu `{ifset blockname}` zkontrolujeme, zda v aktuálním kontextu blok
 {/ifset}
 ```
 
-Název bloku může být jakýkoli platný výraz PHP. Musíte pouze přidat klíčové slovo `block`, aby bylo jasné, že nejde o test existence [proměnných|tags#ifset-elseifset]:
+Název bloku může být jakýkoli platný výraz PHP. Můžete také přidat klíčové slovo `block`, aby bylo jasné, že nejde o kontrolu existence [proměnné|tags#ifset-elseifset]:
```

Comment:
To se opravu musí. Co třeba „V takovém případě musíte přidat…“

Reply:
Jo takhle!! Až teď mi došlo, že se to váže k té proměnné.. Když je proměnná, musí být, jinak být nemusí. To by chtělo podle mě líp vysvětlit, vůbec jsem to nepochopil. Spíš jsem přemýšlel nad tím, proč tam jednou `block` je a jednou ne – [viz fórum](https://forum.nette.org/en/34014-template-inheritance-docs-thanks-and-notes). Možná by si to zasloužilo uvést `{include block}` už výše a pak tady jen zmínit, že s proměnnou je to povinné?

Reply:
Ono to existuje vlastně jen kvůli použití s proměnnou, proto je to uvedené dohromady. Ale budu rád, když to zkusíš vysvětlit líp.

Reply:
Aha, to je fór. Tenhle text se totiž v docs objevuje dvakrát jen s lehkou variací a já to změnil na špatném místě. 🙂 

Měl jsem na mysli `{include block $xyz}`. Tam je to teda jak? Proč tam musí být to `block`? Bez proměnné to má přece taky stejnou syntaxi: `{include footer}` vs `{include 'footer.latte'}`..

Reply:
Jasně, tohle je o `{ifset}`. Tady je asi jasné, proč `block` je nutnost.

S `{include}` je to potřeba taky už v okamžiku kompilace vědět, jestli je to blok nebo soubor, protože se k tomu generuje jiný, jsou tam i jiné možnosti, jako že u souboru může být `{include file.latte with blocks}`, naopak u bloku může být `{include sidebar from file.latte}`, no a když je tam `{include $var ...}` tak on v době kompilace neví, kterou cestou jít.